### DISCUSSION TITLE
Where is github pages that can deploy portfolio page (it suddenly disappeared)

### DISCUSSION THREAD

**Discussion by sinanazeri** [community/community#119702] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/121966646/255ae54b-c235-4429-8117-ed23e8d3f9bf)

Could you help me understand why my GitHub Pages site suddenly became inaccessible? I checked under "Settings" then "Pages" on GitHub, but it shows nothing.

**Comment by LiteBrite82** [comment, marked as answer]
Thanks for posting in the GitHub Community, @sinanazeri !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Pages category. I've gone ahead and moved it for you.
